server/s3_get_object_action.cc

```diff
@@ -132,6 +132,22 @@ void S3GetObjectAction::fetch_object_info_failed() {
 
 void S3GetObjectAction::validate_object_info() {
```

Comment:
from where we r invoking this method

Reply:
This is one of the object action tasks, will be invoked using next() call from fetch_object_info_success().

Reply:
ok